src/Components/Dialogs/Gameplay/Clan/clan-list.scss

```diff
@@ -0,0 +1,144 @@
+.clan-list-dialog {
+  display: flex;
+  flex-direction: column;
+  min-width: 580px;
+  max-width: 680px;
+
+  .title {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+
+    h2 {
+      text-align: center;
+    }
+
+    img {
+      height: 100%;
+      width: 40px;
+      margin-right: 1rem;
+    }
+  }
+
+  .frame-sep{
+    height: 18px;
+    width: 100%;
+
+    .small {
+      height: 10px;
+    }
+  }
+
+
+  .list-box {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: 2%;
+      
+    .filters {
+      display: flex;
+      flex-direction: row;
+      height: 33px;
+      gap: 1rem;
```

Comment:
we cannot use gap with flex in the game engine, this will break, can you implement this without gap?